src/Options.jsx

```diff
@@ -20,6 +20,7 @@ class Options extends React.Component {
     const prefixCls = `${props.rootPrefixCls}-options`;
     const changeSize = props.changeSize;
     const quickGo = props.quickGo;
+    const buildOptionText = this.props.buildOptionText || this.buildOptionText;
```

Comment:
`this.props` was aliased as `props`

custom methods should better with an underscore prefix